sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -108,5 +110,16 @@ internal static DateTimeOffset FormatUtcTimestamp(System.DateTime utcTimestamp)
         {
             return DateTime.SpecifyKind(utcTimestamp, DateTimeKind.Utc);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void SetAuthenticatedUserId(ref ActivityTagsProcessor activityTagsProcessor)
+        {
+            var endUserId = AzMonList.GetTagValue(ref activityTagsProcessor.MappedTags, SemanticConventions.AttributeEnduserId)?.ToString();
+
+            if (endUserId != null)
```

Comment:
This code is on the hot path. Should we add something like [`HasAzureNamespace`](
https://github.com/Azure/azure-sdk-for-net/blob/8a0a766d942a11355c2d48b6f17f6db37776cda1/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ActivityTagsProcessor.cs#L70) to avoid looping through the list?

Reply:
I don't mind making the change, but how are we deciding when a tag should have a bool field?
I see a few other GetTagValue calls in this same class.

Reply:
In this case should we NOT add it to the MappedTags list?